contracts/evoting/evoting.go

```diff
@@ -820,42 +816,38 @@ func (e evotingCommand) manageAdminList(snap store.Snapshot, step execution.Step
 			// Therefore return
 			return nil
 		}
-
-		isAdmin, err := e.isAdmin(form, txAddAdmin.PerformingUserID)
-		if err != nil {
-			return err
-		}
 		if !isAdmin {
 			return xerrors.Errorf("The performing user is not an admin.")
 		}
 
-		err = form.AddAdmin(txAddAdmin.TargetUserID)
+		err = list.AddAdmin(txAddAdmin.TargetUserID)
 		if err != nil {
 			return xerrors.Errorf("couldn't add admin: %v", err)
 		}
 	} else if okRemoveAdmin {
-		form, err = e.getAdminList(snap)
+		list, err = e.getAdminList(snap)
 		if err != nil {
 			return xerrors.Errorf("failed to get AdminList: %v", err)
 		}
```

Comment:
with the changes to `is_admin` this part of the code is no longer necessary (as this will be checked in `is_admin` too)